api_app/analyzers_manager/file_analyzers/pe_info.py

```diff
@@ -71,6 +161,10 @@ def run(self):
                 pe.OPTIONAL_HEADER.MinorOperatingSystemVersion,
             )
 
+            winpe_hasher = WinPEhasher(self.filepath, self)
+            results["dhashicon_hash"] = winpe_hasher.dhashicon()
+            results["impfuzzy_hash"] = winpe_hasher.impfuzzy()
+
```

Comment:
ok custom exceptions but they are not catched anywhere. They should be catched here. Otherwise, as I have already mentioned, if one of this function crashes, all the anaalyzer does crash

Reply:
ok sir. I have one doubt, in case we have a custom exception do we throw it or do we pass it and continue rest of the analysis?

Reply:
we log the error and we continue. As mentioned multiple times, this new code **cannot** change the previous behavior of this analyzer

Reply:
Ok, Sir. I understand now.

Reply:
Hello Sir, Please review the changes :)